nexus/Freqlog/Freqlog.py

```diff
@@ -310,21 +323,19 @@ def __init__(self, backend_path: str, password_callback: callable, loggable: boo
 
         self.backend: Backend = SQLiteBackend(backend_path, password_callback, upgrade_callback)
         self.q: Queue = Queue()
-        self.listener: kbd.Listener | None = None
-        self.mouse_listener: mouse.Listener | None = None
-        if loggable:
-            self.listener = kbd.Listener(on_press=self._on_press, on_release=self._on_release, name="Keyboard Listener")
-            self.mouse_listener = mouse.Listener(on_click=self._on_click, name="Mouse Listener")
+        self.listener = None
         self.new_word_threshold: float = Defaults.DEFAULT_NEW_WORD_THRESHOLD
         self.chord_char_threshold: int = Defaults.DEFAULT_CHORD_CHAR_THRESHOLD
         self.allowed_chars: set = Defaults.DEFAULT_ALLOWED_CHARS
         self.allowed_first_chars: set = Defaults.DEFAULT_ALLOWED_FIRST_CHARS
-        self.modifier_keys: set = Defaults.DEFAULT_MODIFIER_KEYS
+        self.modifier_keys: vinput.KeyboardModifiers = vinput.KeyboardModifiers()
+        for attr in Defaults.DEFAULT_MODIFIERS:
+            setattr(self.modifier_keys, attr, True)
         self.killed: bool = False
 
     def start_logging(self, new_word_threshold: float | None = None, chord_char_threshold: int | None = None,
                       allowed_chars: set | str | None = None, allowed_first_chars: set | str | None = None,
-                      modifier_keys: set = None) -> None:
+                      modifier_keys: vinput.KeyboardModifiers = vinput.KeyboardModifiers()) -> None:
```

Comment:
This needs to be `= None` by default for the `if modifier_keys is not None: self.modifier_keys = modifier_keys` below to work.